packages/react/src/utils/useScrollLock.ts

```diff
@@ -167,7 +167,9 @@ function preventScrollInsetScrollbars(referenceElement: Element | null) {
   return () => {
     resizeFrame.cancel();
     cleanup();
-    win.removeEventListener('resize', handleResize);
+    if (win && win.removeEventListener) {
```

Comment:
```suggestion
    // Sometimes this cleanup can be run after test teardown 
    // because it is called in a `setTimeout(fn, 0)`,
    // in which case `removeEventListener` wouldn't be available,
    // so we check for it to avoid test failures.
    if (win.removeEventListener) {
```